tests/tracking/_model_registry/test_utils.py

```diff
@@ -123,6 +131,13 @@ def test_get_store_caches_on_store_uri(tmpdir):
     assert store1 is not store3
 
 
+@pytest.mark.parametrize("store_uri", ["databricks-uc", "databricks-uc://profile"])
+def test_get_store_raises_on_uc_registry_uri(store_uri, reset_registry_uri):
+    set_registry_uri(store_uri)
+    with pytest.raises(MlflowException, match="does not support models in the Unity Catalog"):
+        mlflow.tracking.MlflowClient().search_registered_models()
```

Comment:
Which function call throws an exception, `MlflowClient()` or `search_registered_models()`?

Reply:
```suggestion
        mlflow.tracking.MlflowClient()
```

If the former, can we remove `search_registered_models`?

Reply:
Good question it's the latter, seems we only construct the model registry store when we actually call a registry client method. I'll pull the client initialization out of the `with pytest.raises` block :)

Reply:
> seems we only construct the model registry store when we actually call a registry client method

Didn't know that. Thanks for the update!